components/Nav.tsx

```diff
@@ -1,46 +1,40 @@
 import React from 'react';
 import useArconnect from 'use-arconnect';
 import Link from 'next/link';
-import { PermissionType } from 'arconnect';
+import Router from 'next/router';
+import useGetUser from '../hooks/useGetUser';
 
 const Nav = () => {
-  const arconnect = useArconnect();
-
-  const arConnectPermissions: PermissionType[] = [
-    'ACCESS_ADDRESS',
-    'ACCESS_ALL_ADDRESSES',
-    'SIGN_TRANSACTION',
-  ];
-
-  async function login() {
-    await window.arweaveWallet.connect(arConnectPermissions, {
-      name: 'CommunityLabs News',
-    });
-  }
+  const arconnect = useArconnect(),
+    { currentUser } = useGetUser(),
+    navItems: Array<'new' | 'threads' | 'comments' | 'submit'> = [
```

Comment:
Make this a type
```
type navItems = ['new', 'threads', 'comments', 'submit]
```
But rather than type
I would strongly encourage use enums. That's what enums are for.

> An enumeration, or Enum , is a symbolic name for a set of values. Enumerations are treated as data types, and you can use them to create sets of constants for use with variables and properties

> In [TypeScript](https://www.typescriptlang.org/), enums, or enumerated types, are data structures of constant length that hold a set of constant values. Each of these constant values is known as a member of the enum. Enums are useful when setting properties or values that can only be a certain number of possible values